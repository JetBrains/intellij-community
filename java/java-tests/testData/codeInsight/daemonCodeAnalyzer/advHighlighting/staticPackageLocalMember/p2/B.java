package p2;

import p1.A;

public class B extends A {
}
