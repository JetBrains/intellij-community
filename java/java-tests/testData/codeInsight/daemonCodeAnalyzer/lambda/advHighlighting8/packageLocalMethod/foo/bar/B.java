package foo.bar;
import foo.A;
abstract class B extends A {}