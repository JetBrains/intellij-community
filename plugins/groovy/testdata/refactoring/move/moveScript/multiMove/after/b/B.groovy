package b

class B {
  def fooB(int i) {
    Script x;
    return i+1;
  }
}