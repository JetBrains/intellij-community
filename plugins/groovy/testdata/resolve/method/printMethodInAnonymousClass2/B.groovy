new Runnable() {
  void run() {
    pri<caret>nt "wow";
  }
}

def print(String s) {}