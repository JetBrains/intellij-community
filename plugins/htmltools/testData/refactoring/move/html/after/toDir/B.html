<a href="../A.html">anchor</a>