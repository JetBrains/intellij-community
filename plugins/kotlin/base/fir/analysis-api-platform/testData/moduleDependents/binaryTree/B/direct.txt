A (production)
A (test)
B (test)
