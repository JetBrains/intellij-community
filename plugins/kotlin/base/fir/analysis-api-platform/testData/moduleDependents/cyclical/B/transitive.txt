A (production)
A (test)
B (test)
C (production)
C (test)
