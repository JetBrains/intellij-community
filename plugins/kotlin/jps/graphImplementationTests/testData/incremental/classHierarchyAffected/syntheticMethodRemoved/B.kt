package foo

open class B {
  interface Presentation {
    var text: String
  }
}