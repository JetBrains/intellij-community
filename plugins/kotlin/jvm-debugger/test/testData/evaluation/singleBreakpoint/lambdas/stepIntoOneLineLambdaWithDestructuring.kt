// ATTACH_LIBRARY: utils
package stepIntoOneLineLambdaWithDestructuring

import destructurableClasses.B

fun foo(f: (B) -> Unit) {
    val b = B()
    // STEP_INTO: 1
    // EXPRESSION: x + y
    // RESULT: 2: I
    //Breakpoint!
    f(b)
}

fun main() {
    foo { (x, _, y, _) -> println() }
}

// PRINT_FRAME
