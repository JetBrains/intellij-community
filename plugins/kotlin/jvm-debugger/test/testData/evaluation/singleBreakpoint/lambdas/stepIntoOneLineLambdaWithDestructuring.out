// IGNORE_BACKEND: JVM_WITH_OLD_EVALUATOR
LineBreakpoint created at stepIntoOneLineLambdaWithDestructuring.kt:12
Run Java
Connected to the target VM
stepIntoOneLineLambdaWithDestructuring.kt:12
stepIntoOneLineLambdaWithDestructuring.kt:16
Compile bytecode for x + y
KotlinStackFrame (stepIntoOneLineLambdaWithDestructuring.kt:16)
    JavaValue[this] this: StepIntoOneLineLambdaWithDestructuringKt$main$1@uniqueID = Function1<destructurableClasses.B, kotlin.Unit>
        JavaValue[field] arity: int = 1 (Lambda.!EXT!)
    JavaValue[local] x: int = 1 (stepIntoOneLineLambdaWithDestructuring.kt:16)
    JavaValue[local] y: int = 1 (stepIntoOneLineLambdaWithDestructuring.kt:16)
Disconnected from the target VM

Process finished with exit code 0
